var/spack/repos/builtin/packages/py-pillow/package.py

```diff
@@ -25,6 +25,7 @@ class PyPillowBase(PythonPackage):
     variant("jpeg2000", default=False, description="JPEG 2000 functionality")
     variant("imagequant", when="@3.3:", default=False, description="Improved color quantization")
     variant("xcb", when="@7.1:", default=False, description="X11 screengrab support")
+    variant('raqm', default=False, description='RAQM support')
```

Comment:
Any idea what the first version of pillow supporting RAQM is?

Reply:
Good question. Version 8.4.0 says `error in setup.cfg: command 'pil_build_ext' has no such option 'enable_raqm'`, the next version in Spack 9.0.0 works.

Reply:
Fixed in 53740ac22bdf7bc859ec8947f8bd0a594587ad31.